runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -330,6 +352,12 @@ private void addReconnectionOnCloseFuture(@Nonnull Channel channel,
             // If we aren't shutdown, reconnect.
             if (!shutdown) {
                 log.info("addReconnectionOnCloseFuture[{}]: reconnecting", node);
+                if (sslAuthenticationFailure.get()) {
+                    retryPolicy.nextWait();
```

Comment:
Can we make nextWait return the next value, so that we can invoke the sleep logic directly? 
That way we can add logging with the right context.

Reply:
Done